app/service/ccd-service.ts

```diff
@@ -92,12 +115,16 @@ class CcdService {
     );
   }
 
-  retrieveCaseHistory(userId: string, headers: SecurityHeaders, caseId: string): Promise<CcdCaseDetails[]> {
-    return rp.get(this.createOptions(
+  retrieveCaseHistoryV2(userId: string, caseId: string, headers: SecurityHeaders): Promise<any> {
```

Comment:
Might be better to leave it as any for now as this is only a BETA endpoint it could change anytime. But we should keep in mind to do this once we know the response is stable.